### PR TITLE
Board page

### PR DESCRIPTION
**Pull request by Pymossy**

### CODE REVIEW

File: src/assets/icons/Done.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import DoneIcon from './done.png';
+
+interface OwnProps {}
+
+export function Done(props: OwnProps) {
+  return <img {...props} alt="done icon" src={DoneIcon} />;
```
Comment (smmoosavi) [inline src/assets/icons/Done.tsx]: use i18n for alt prop,

```suggestion
  return <img {...props} alt={i18n._('Done')} src={DoneIcon} />;
```

File: src/assets/icons/InProgress.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import InProgressIcon from './in-progress.png';
+
+interface OwnProps {}
+
+export function InProgress(props: OwnProps) {
+  return <img {...props} alt="in progress icon" src={InProgressIcon} />;
```
Comment (smmoosavi) [inline src/assets/icons/InProgress.tsx]:
```suggestion
  return <img {...props} alt={i18n._('In Progress')} src={DoneIcon} />;
```

File: src/assets/icons/Todo.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import TodoIcon from './to-do.png';
+
+interface OwnProps {}
+
+export function Todo(props: OwnProps) {
+  return <img {...props} alt="todo icon" src={TodoIcon} />;
```
Comment (smmoosavi) [inline src/assets/icons/Todo.tsx]: return ;

File: src/pages/peer-review-board-page/EmptyList.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { makeStyles, Theme, Typography } from '@material-ui/core';
+import { FCProps } from 'src/shared/types/FCProps';
+import { PlaceHolder } from 'src/shared/board-list';
+import { Styles } from 'src/shared/types/Styles';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
```
Comment (sijav) [inline src/pages/peer-review-board-page/EmptyList.tsx]: order imports

File: src/locales/en/messages.po [hunk 5]
```diff
@@ -30,6 +31,10 @@ msgstr "Achievements"
 msgid "Add"
 msgstr "Add"
 
+#: src/pages/peer-review-board-page/PeerReviewBoardPage.tsx:58
+msgid "Add every review that completed here"
```
[the remark this answers is not in the document]
Comment (sijav) [reply src/locales/en/messages.po]:
```suggestion
msgid "Add here after completing any evaluation"
```

File: src/locales/en/messages.po [hunk 6]
```diff
@@ -379,6 +405,10 @@ msgstr "Yes"
 msgid "Yes! Go"
 msgstr "Yes! Go"
 
+#: src/pages/peer-review-board-page/PeerReviewBoardPage.tsx:53
+msgid "You don't start to review yet!"
```
[the remark this answers is not in the document]
Comment (sijav) [reply src/locales/en/messages.po]:
```suggestion
msgid "You haven't started evaluating yet!"
```

File: src/locales/en/messages.po [hunk 7]
```diff
@@ -379,6 +405,10 @@ msgstr "Yes"
...
+msgstr "You don't start to review yet!"
+
 #: src/pages/dashboard-page/AchievementsNoProjects.tsx:6
 msgid "You haven't added any project."
```
[the remark this answers is not in the document]
Comment (sijav) [reply src/locales/en/messages.po]:
```suggestion
msgid "You haven't added any projects yet."
```

**Pull request merged**